reporting/report.go

```diff
@@ -41,6 +41,11 @@ func (r *Report) ShowReport(format string, cfg *config.Config) error {
 
 func (r *Report) WriteFile(reportPath []string, cfg *config.Config) error {
 	for _, path := range reportPath {
+		slashPath := filepath.ToSlash(path)
+		if _, err := os.Stat(filepath.Dir(slashPath)); os.IsNotExist(err) {
+			os.MkdirAll(filepath.Dir(slashPath), 0600)
+		}
```

Comment:
From `MkdirAll` docs:

> If path is already a directory, MkdirAll does nothing and returns nil.

I think you don't need to check `IsNotExist`, you can immediately use the `MkdirAll(Dir(slashPath))`.
With this you also not ignoring the `err` when it is other than `IsNotExist`.

Reply:
Ah, yes! Good catch. Will update.